src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -446,7 +446,7 @@ public function testGetFormatFromMimeType($format, $mimeTypes)
     public function getFormatToMimeTypeMapProviderWithAdditionalNullFormat()
     {
         return array_merge(
-            [[null, [null, 'unexistent-mime-type']]],
+            [[null, ['unexistent-mime-type']]],
```

Comment:
`Request::getFormat()` does not accept `null`:

https://github.com/symfony/symfony/blob/b306bdb5244fc2b00726b70f9e2fcf4e76420dda/src/Symfony/Component/HttpFoundation/Request.php#L1308-L1315